beep/dataset.py

```diff
@@ -339,6 +340,104 @@ def serialize(self):
         return self.dataset_dir
 
 
+def get_threshold_targets(dataset_diagnostic_properties,
+                          cycle_type_target="rpt_1C",
+                          basis="discharge_energy",
```

Comment:
Again, for the sake of consistency with other modules and method, call `basis` ->`metric`?

Reply:
yup, done!